src/descriptor/policy.rs

```diff
@@ -108,39 +108,60 @@ impl PKOrF {
 #[serde(tag = "type", rename_all = "UPPERCASE")]
 pub enum SatisfiableItem {
     // Leaves
+    /// Signature for a raw public key
     Signature(PKOrF),
+    /// Signature for an extended key fingerprint
     SignatureKey(PKOrF),
+    /// SHA256 preimage hash
     SHA256Preimage {
+        #[allow(missing_docs)]
```

Comment:
If the decision was made by the project to enforce documentation then should we be erring on the side of too much documentation instead of explicitly allowing no docs? Seems cleaner to me to add the extra rustdoc.

For example 
```
    /// RIPEMD160 preimage hash
    RIPEMD160Preimage {
        /// The digest value
        hash: ripemd160::Hash,
    },
```

Reply:
I was trying to avoid being redundant with what's already in the struct doc, but I think you're right, adding extra doesn't hurt. I just added for the descriptor module.